app/routers/weekview.py

```diff
@@ -22,7 +22,8 @@
 class DayEventsAndAttrs(NamedTuple):
     day: datetime
     template: Jinja2Templates.TemplateResponse
-    events_and_attrs: Tuple[Event, DivAttributes]
+    events_and_attrs: Tuple[Event, EventsAttributes]
+    current_time_and_attrs: CurrentTimeAttributes
```

Comment:
It's a bit fuzzy - what do you mean by `and_attrs`?

Reply:
Actually, I just followed the existing name events_and_attr (as for attributes) and created current_time_and_attrs. It is the current time with some extra details attached to by the CurrentTimeAttributes class. (for example the grid position of the current time etc)